test/index.js

```diff
@@ -41,3 +41,10 @@ if(color === 'red') {
 }
 
 typeof color === 'strnig';
```

Comment:
string? 😁

Reply:
it is test code. I added a positive and negative test here: https://github.com/digitalbazaar/eslint-config-digitalbazaar/pull/44/commits/dc68ce247aad37489faf235dc840da933d9d1da5